instances/technique/eyeMovementTracking.jsonld

```diff
@@ -4,11 +4,14 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/eyeMovementTracking",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "'Eye movement tracking' is a technique to measure the eye movement and/or position to determine what an individual was focusing on at a given time.",
```

Comment:
I would also refer this to a group of techniques. There is more than one to track eye movement and keep it general (without the purpose).

"'Eye movement tracking' refers to a group of techniques used to measure over a given period of time the eye movement and/or position of a living specimen."

Reply:
Ok, change it